services/web/server/src/simcore_service_webserver/products/_api.py

```diff
@@ -28,6 +29,17 @@ def list_products(app: web.Application) -> list[Product]:
     return products
 
 
+async def get_current_product_price(request: web.Request) -> ProductPriceGet:
```

Comment:
very minor: to be picky, I find a bit weird that you use a handler model in the api code.

Reply:
changed